fpsim/locations/senegal/data/contra_coef_mid.csv

```diff
@@ -0,0 +1,13 @@
+"Estimate","Std. Error","t value","Pr(>|t|)","rhs"
+-20.5661824094816,0.152524562781688,-134.838494432654,9.25206040808785e-179,"(Intercept)"
```

Comment:
I've been running sims with these coefficients and I think we may want to reconsider initialization if we're going with thesevalues. These values imply that someone without prior contraceptive use has a probability of ~5e-9 of using contraception. Everyone in the simulations start out as a non-user, so we basically never get any users with these parameters. We could either initialize some women as previous users, or we could find a way to re-estimate these values.

Reply:
Looking at this now. I might need to brainstorm about the with @MObrien-IDM . I'm surprised that coefficient is so large but I see why it's an issue. Let me think about it for today.